File: Proyecto/Arbol infinito/core/nodo.py
class nodo:
	identificador = None
	padre = None
	apuntadores = []


	def __init__(self,identificador,padre):
		self.apuntadores = []
		self.identificador = identificador
		self.padre = padre


	@staticmethod
    	def serialize(obj):
    		hijo={}
    		print("---------------------------obj------------------------------------")
    		print(obj)
    		print(type(obj))
    		print(obj.identificador)
    		if(len(obj.apuntadores) != 0):
    			print("------------------------------hijos-------------------------------")
    			print(obj.apuntadores)
    			print(type(obj.apuntadores))
    			for item in obj.apuntadores:
    				print("------------------item-------------------------------------------")
    				print(item)
    				print(type(item))
    				print(item.identificador)
    				hijo = obj.serialize(item)
    		
    		

		return {
		   "dic "+item.identificador:
			item.identificador,
						"hijos "+item.identificador:[
						Arbol.serialize(item)
						]	
		}




	def getApuntador(self, valor):
		"""if [valor] in self.apuntadores:
			return self.apuntadores.index(valor)
		else:
			return None"""

		for item in self.apuntadores:
			if item.identificador == valor:
				return item


	def getId(self):
		
		return self.identificador


	def setApuntador(self, valor):
		
		self.apuntadores.append(valor)

	def setId(self, identificador):
		
		self.identificador = identificador

File: Proyecto/Arbol Infinito JS/core/js/clases/dtas/Nodo.js
export default class Nodo{

	constructor(identificador,padre){
		console.log("Soy un nodo");
		this.identificador = identificador;
		this.padre = padre;
		this.apuntadores = [];
	}

	getHijo(identificador){
		var respuesta = null;
		this.apuntadores.forEach(function(item){
			if(item.identificador == identificador){
				console.log("se encontro");
				console.log(item);
				respuesta = item;
			}
		});
		return respuesta;

	}

	setHijo(nodo){
		this.apuntadores.push(nodo);
	}
}